gutenberg/blocks/ts/src/components/drag-and-drop-select-list.tsx

```diff
@@ -59,36 +73,48 @@ class DragAndDropSelectList extends React.Component<Props, State> {
   draggedItem: any;
   draggedIdx: any;
 
-  onDragStart = (e: any, index: number) => {
+    /**
+   * Event handler for drag start 
+   * 
+   * @param e 
+   * @param index items index on selected list
+   */
+  private onDragStart = (e: any, index: number) => {
```

Comment:
Methods is class should be arranged in following order:

public, protected, private, private event handlers